### PR TITLE
Add NuGet v3 feed support

### PR DESCRIPTION
**Pull request by ChengTian**
@loudej  @davidfowl  @muratg  @yishaigalatzer

### CODE REVIEW

File: src/Microsoft.Framework.PackageManager/Restore/NuGet/HttpSource.cs [hunk 1]
```diff
@@ -131,13 +139,20 @@ internal class HttpSource
 
                 using (var response = await _client.SendAsync(request, HttpCompletionOption.ResponseHeadersRead))
                 {
+                    if (!throwNotFound && response.StatusCode == HttpStatusCode.NotFound)
```
Comment (ChengTian) [inline src/Microsoft.Framework.PackageManager/Restore/NuGet/HttpSource.cs]: For v3 feed, FindPackagesById() becomes construct the URI by ourselves and check whether the URI points to a JSON. A 404 page means the packages is not on this feed.

**Comment by davidfowl** [comment]
@ChengTian wrong branch?

File: src/Microsoft.Framework.PackageManager/Restore/NuGet/HttpSource.cs (hunk 1 above)
Comment (yishaigalatzer) [inline src/Microsoft.Framework.PackageManager/Restore/NuGet/HttpSource.cs]: Where is the change to change the URL from v3 to v2 in the beta5 branch?

-----Original Message-----

Comment (davidfowl) [inline src/Microsoft.Framework.PackageManager/Restore/NuGet/HttpSource.cs]: We're doing the real change in release and dev

Comment (yishaigalatzer) [inline src/Microsoft.Framework.PackageManager/Restore/NuGet/HttpSource.cs]: Cool

-----Original Message-----

**Comment by ChengTian** [comment]
Just wrong target branch, the changes should be correct. Will create a new PR with correct target branch though

**Pull request closed without merging**